about-us/index.html

```diff
@@ -42,29 +42,28 @@
         <p class="text-justify">
             <br>
             <br>
-            UMWICS exists to support the female Computer Science students at the University of Manitoba and we aim
-            to promote awareness of the importance of women within the Computer Science community.
+            WICS is here to build a stronger community between both women and men in STEM,
+            as well as to encourage more women into the computer science field!
 
             <br>
             <br>
-            We have done a project in web-development using HTML, JavaScript, and CSS, as well as a project in
-            application development using Android Studio, and are looking into making many
-            more projects in different languages in the future.
-
-            <br>
+            We are constantly working on projects outside of the classroom and have shown major interest in web and
+            application development.
+            In fact, this very website was created by our WICS members using HTML, JavaScript,
+            CSS, and Bootstrap!
             <br>
-            We have organized a Hack Day this past November, and look forward to organizing another Hack Day this year
-            where we will spend the day enhancing our coding skills.
-
             <br>
+            We have also put a main focus on encouraging elementary and junior high students into the Computer Science field and
+            have been using the website
+            <i><a href="https://hourofcode.com/ca/learn" class="black-link" target="black">Hour of Code</a></i>
+            to help kids gain the skills to already start programming on their own!
+            We have already visited Windsor Park Elementary School and Carman Collegiate and all other
```

Comment:
"...and all other schools are welcome us to contact us by sending us an email at..."
This sentence does not make sense.

Consider rephrasing to: "...and other schools are welcome to contact us by email at..."
Other than that, it looks great (provided the school names are correct as Julia pointed out).
👍

Reply:
Thanks! Just changed it